rsync-formula/rsync/init.sls

```diff
@@ -70,3 +70,14 @@ rsyncd_service:
     - watch:
       - file: rsyncd_config
     {%- endif %}
+
+{%- for module, module_config in config.get('modules', {}).items() %}
```

Comment:
We're using 5 jinja lines to ensure that we don't install a simple script everywhere. I would install everywhere or add a comment saying that this is only aesthetic without critical logic.